File: backend/.NET/Calindra.Desafio/Calindra.Desafio.WebApi/Models/ErrorModel.cs
namespace Calindra.Desafio.WebApi.Models
{
    public class ErrorModel
    {
        public ErrorModel() { }

        public ErrorModel(int statusCode, string message, object details = null)
        {
            StatusCode = statusCode;
            Message = message;
            Data = details;
        }

        public int StatusCode { get; set; }
        public string Message { get; set; }
        public object Data { get; set; }
    }
}

File: backend/Node.JS/calindra-desafio/src/controllers/v1.0/geolocation/models/error/error-model.ts
export class ErrorModel {

    constructor(statusCode: number, message: string | null, error: string, details: any = null) {
        this.statuCode = statusCode;
        this.message = message;
        this.error = error;
        this.details = details;
    }

    public statuCode: number;
    public message: string | null;
    public error: string;
    public details: any;
}